app/models/training/content.rb

```diff
@@ -16,6 +16,16 @@ def parent
       @parent ||= Training::Module.by_content_id(id)
     end
 
+    # @return [Boolean]
+    def updated_content?
+      created_at < updated_at
```

Comment:
How is this affected by the module publish date? Also, is there an alternative predicate name that is more appropriate?

Reply:
This has been updated to include the published at update